global_body_planner/global_body_planner.yaml

```diff
@@ -1,31 +1,35 @@
 global_body_planner:
-  update_rate: 10
+  update_rate: 20
   num_calls: 1            # Number of times to call the planner
-  max_planning_time: 2.0  # Minimum amount of time to run the planner (will replan to find better solutions until this duration is up)
+  max_planning_time: 0.05  # Maximum amount of time to run the planner
```

Comment:
Isnt this a bit too low?

Reply:
Updated but should be fine for now, updates to handle difficult plans will come later (and it will still return a partial plan if it can't find one in time).